lib/cog/commands/bundle.ex

```diff
@@ -19,14 +19,10 @@ defmodule Cog.Commands.Bundle do
     "info <bundle>" => "Detailed information on an installed bundle",
     "enable <bundle> [<version>]" => "Enable a specific version of an installed bundle",
     "disable <bundle>" => "Disable an installed bundle",
-    "versions <bundle>" => "List all installed versions for a given bundle"
+    "versions <bundle>" => "List all installed versions for a given bundle",
+    "install <bundle>[:<version>]" => "Install latest or specified version of bundle from registry"
   }
 
-  @notes """
-  Installation and uninstallation of bundles cannot currently be done via
-  chat; please use `cogctl` for this functionality.
-  """
-
   permission "manage_commands"
 
   rule "when command is #{Cog.Util.Misc.embedded_bundle}:bundle must have #{Cog.Util.Misc.embedded_bundle}:manage_commands"
```

Comment:
Somewhat tangential to this PR, but we should probably consider adding some `allow` rules for the `list`, `info`, and `versions` subcommands.

(Locking down installation of bundles, of course, is definitely the way to go, though 😄 )